frontend/src/pages/home/aiFlows/ProjectsGallery.tsx

```diff
@@ -24,7 +51,16 @@ const ProjectsGallery: React.FC<{ onClose: () => void }> = ({ onClose }) => {
       title="Projects"
       imgSrc={typedObjectImage(ProjectObjectType.project)}
       sectionType={SectionType.organize}
-      description={projectDescription}
+      description={
+        <span>
+          Projects allow you and your team to organize and collaborate on resources within separate
+          namespaces.
+          <br />
```

Comment:
@jgiardino It is already in a `<small>`, would the `<br>` not be sized as one line height of the small? I can change it to be specified by every caller of the `<InfoGalleryItem>` rather than in the common component.

Reply:
The rule-of-thumb I've always followed is to use separate `<p>` tags for separate paragraphs to get the preferred spacing between paragraphs. In this case, the space between would be slightly different if implemented as separate `<small>` elements vs using `<br>`. TBH, I'm not sure if there are any a11y implications with using line breaks vs separate `<small>` elements, so it may just be a cosmetic issue.

Reply:
OK, I've updated it to use multiple `<small>` components.